scripts/generate_user_permissions.py

```diff
@@ -0,0 +1,117 @@
+#--------------------------------------------------------------------------------
+# Run this script as follow, it can only read from Google Docs
+#
+# Requires packages:
+#                   1. gspread
+#                   2. oauth2client
+#
+# python generate_user_permissions.py -f  "oath_credentials.json" 
+#                                     -ss "google_user_spreadsheet_permissions"
+#                                     -ss "google_bot_spreadsheet_permissions"
+#--------------------------------------------------------------------------------
+import json
+import sys 
+import os 
+import argparse
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("-f", "--oauth_credentials_file", help="Path of the file where oauth credetials to " +
+                    "access google drive API is stored, Make sure the email in this file " + 
+                    "has edit access on the spreadsheets being read in this script")
+    parser.add_argument('-ss', "--spreadsheet_title", action='append')
+    parser.add_argument('-p', '--password_column_name', help="Name of the column in the spreadsheet which has password,"+
```

Comment:
minor - space after comma - `password,`
question - Why not make this default to "username" or even make it mandatory so there is not explicit default logic in code?